src/components/container/sticky-bar/sticky-bar.scss

```diff
@@ -0,0 +1,38 @@
+@use '../../core/styles' as sbb;
+
+:host {
+  @include sbb.box-sizing;
+
+  --sbb-sticky-bar-padding-block: var(--sbb-spacing-responsive-xxxs);
+  --sbb-sticky-bar-animation-duration: var(--sbb-animation-duration-5x);
+
+  display: contents;
+}
+
+.sbb-sticky-bar__wrapper {
+  box-shadow: none;
+  position: sticky;
+  inset-block-end: 0;
+  display: inline-block;
+  width: 100%;
+  background-color: var(--sbb-container-background-color, var(--sbb-color-white-default));
+  transition-property: box-shadow;
+  transition-duration: var(--sbb-sticky-bar-animation-duration);
```

Comment:
transition-timing-function missing. maybe try with var(--sbb-animation-easing) and introduce --sbb-sticky-bar-animation-easing var